clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelPropertiesTableRow.tsx

```diff
@@ -49,6 +49,23 @@ const ModelPropertiesTableRow: React.FC<ModelPropertiesTableRowProps> = ({
 
   const [isValueExpanded, setIsValueExpanded] = React.useState(false);
 
+  const [isSwitched, setIsSwitched] = React.useState<boolean>(window.isSwitched ?? false);
+
+  // Listen for changes to `window.isSwitched`
+  React.useEffect(() => {
+    const handleSwitchChange = (event: Event) => {
+      if (event instanceof CustomEvent) {
+        setIsSwitched(event.detail); // Update state with the new value
+      }
+    };
+
+    window.addEventListener('isSwitchedChanged', handleSwitchChange);
+
+    return () => {
+      window.removeEventListener('isSwitchedChanged', handleSwitchChange);
+    };
+  }, []);
```

Comment:
Since this gets reused a lot I would guess a custom hook like `useMUI` would be a good idea